static/app/components/group/sentryAppExternalIssueActions.tsx

```diff
@@ -37,36 +35,24 @@ type Props = {
   externalIssue?: PlatformExternalIssue;
 };
 
-type State = {
-  action: 'create' | 'link';
-  externalIssue?: PlatformExternalIssue;
-};
-
-class SentryAppExternalIssueActions extends Component<Props, State> {
-  state: State = {
-    action: 'create',
-    externalIssue: this.props.externalIssue,
-  };
-
-  componentDidUpdate(prevProps: Props) {
-    if (this.props.externalIssue !== prevProps.externalIssue) {
-      this.updateExternalIssue(this.props.externalIssue);
-    }
-  }
-
-  updateExternalIssue(externalIssue?: PlatformExternalIssue) {
-    this.setState({externalIssue});
-  }
-
-  doOpenModal = (e?: React.MouseEvent) => {
+function SentryAppExternalIssueActions({
```

Comment:
hiding whitespace can help review this, no longer a class component, no longer has any state